pkg/autoops/api/api.go

```diff
@@ -1010,6 +1049,72 @@ func (s *AutoOpsService) ExecuteAutoOps(
 	return &autoopsproto.ExecuteAutoOpsResponse{AlreadyTriggered: false}, nil
 }
 
+func (s *AutoOpsService) stopProgressiveRolloutIfneeded(
+	ctx context.Context,
+	environmentNamespace string,
+	autoOpsRule *domain.AutoOpsRule,
+	storage v2as.ProgressiveRolloutStorage,
+	localizer locale.Localizer,
+) error {
+	// Stop the running progress rollout if the operation type is disable
+	if autoOpsRule.OpsType == autoopsproto.OpsType_DISABLE_FEATURE {
```

Comment:
We only stop the progressive rollout if the operation type is `DISABLE_FEATURE`.

Reply:
Done.